src/Database/Log/LoggingStatement.php

```diff
@@ -41,6 +41,20 @@ class LoggingStatement extends StatementDecorator
      */
     protected $_compiledParams = [];
 
+    /**
+     * Query execution start time.
+     *
+     * @var float
+     */
+    protected $startTime = 0;
```

Comment:
Not sure how strict the tooling is, but strictly speaking the value would need to be `0.0` to pass as float, and not int.